.gitmodules

```diff
@@ -30,4 +30,10 @@
 	url = https://github.com/google/benchmark
 [submodule "external/alfons"]
 	path = external/alfons
-	url = https://github.com/hjanetzek/alfons
+	url = https://github.com/lygstate/alfons
```

Comment:
Your changes on alfons have been merged on this branch https://github.com/hjanetzek/alfons/tree/wip. All you have to do for this submodule is retarget to it.